core/shared/platform/windows/win_thread.c

```diff
@@ -830,7 +830,8 @@ os_thread_get_stack_boundary()
 
 void
 os_thread_jit_write_protect_np(bool enabled)
-{}
+{
+}
```

Comment:
please don't modify code here, or the CI will report error:
https://github.com/bytecodealliance/wasm-micro-runtime/actions/runs/12782273796/job/35631476862

Reply:
funnily enough, it was clang-format that changed it. I guess it is a different version.